src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExtensions.cs

```diff
@@ -107,17 +108,12 @@ private static double GetMinFromBoundaries(double histogramSum, long histogramVa
 					// the current bucket contains something.
 					if (i == 0)
 					{
-						// If we are in the first bucket, use the upper bound (which is the lowest specified bound
-						// overall) otherwise this would be -Inf, which is not allowed. This is not quite correct,
-						// but the best approximation we can get at this point. This might however lead to a min
-						// that is bigger than the sum, therefore we return the min of the sum and the lowest
-						// bound.
-						// Choose the minimum of the following three:
+						// In the first bucket, (-Inf, firstBound], use firstBound (this is the lowest specified
+						// bound overall). This is not quite correct but the best approximation we can get at this
+						// point. However, this might lead to a min bigger than the mean, thus choose the minimum
+						// of the following:
 						// - The lowest boundary
-						// - The sum (smallest if there are multiple negative measurements smaller than the lowest
-						// boundary)
-						// - The average in the bucket (smallest if there are multiple positive measurements
-						// smaller than the lowest boundary)
+						// - The average of the histogram (histogram sum / sum of counts)
 						return Math.Min(
 							Math.Min(buckets[i].ExplicitBound, histogramSum),
```

Comment:
I dont think we need this anymore, right? We just use the min of bound and mean.

Reply:
```suggestion
							buckets[i].ExplicitBound, 
```

Reply:
Thanks for catching this, I missed that one. Fixed in [510999b](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/32/commits/510999bd24f4058c72db3e9950bbc6a3827abfff).